### PR TITLE
Add temperature change card in site dashboard

### PR DESCRIPTION
**Pull request by K-Markopoulos**
## Issue #1022 

Add 7-days temperature change card in place of Buoy Observation card when site doesn't have a spotter.

- Calculate the change based on `satelliteTemperature` from site's `dailyData`:  avg of 7 last days minus avg of 8-14 days back
- Calculate the avg temp of the last 7 days.

image

### CODE REVIEW

File: packages/website/src/common/SiteDetails/index.tsx [hunk 1]
```diff
@@ -42,7 +43,7 @@ import Map from './Map';
 import SketchFab from './SketchFab';
 import FeaturedMedia from './FeaturedMedia';
 import Satellite from './Satellite';
-import Sensor from './Sensor';
+// import Sensor from './Sensor';
```
Comment (ericboucher) [inline packages/website/src/common/SiteDetails/index.tsx:46]: Note that we still want to display the Sensor card when there is a spotter/buoy with live data for that site

**Comment by Caesarh97** [comment]
Hi @K-Markopoulos,

I think many of the changes look great. However, I'd like to add some more information about the card. We would like the number in color to be calculated by the past 7 days compared to the 7 days before that. As Eric mentioned, this card should be shown if there's no buoy data available. If a site has water quality data and no buoy, we should still display the heat stress card together with the water quality card like before. I think we would want to show the temperature and one decimal number (ex. 22.6°C, just like you've done in the example). 

I think this calculation would be the most accurate to calculate the past 7 days compared to the 7 days before that. 
(sum of most recent 7 days) / 7 = (average temp for past 7 days)
(Sum of past 14-8 days / 7 = (average temp for past 14-8 days)
(average temp for past 7 days) - (average temp for past 14-8 days) = average 7-day change in temperature for the past 7 days compared to the previous 7 days.

Example:
Past 14 days (Day 14 - Day 1. Left to right)
23, 24, 26, 22, 19, 20, 24, 22, 25, 26, 25, 23, 23, 22
(Sum of 22, 25, 26, 25, 23, 23, 22) / 7 = 23.7 (average most recent 7 days)
(Sum of 23, 24, 26, 22, 19, 20, 24) / 7 = 22.6 (average past 14-8 days)
23.7-22.6=1.1
The average 7-day change in temperature for the past 7 days compared to the previous 7 days is 1.1°C.

**Comment by K-Markopoulos** [comment]
@Caesarh97 Thanks for the detailed explanation, I've updated the logic. Please let me know if anything seems off.

**Comment by Caesarh97** [comment]
> the past 7 days compared to the 7 days before that.

@K-Markopoulos That looks great! Thank you!

**Review by ericboucher — APPROVED** [review]

**Pull request merged**